ajax/libs/jqueryrotate/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "jqueryrotate",
+  "filename": "jQueryRotateCompressed.js",
+  "description": "jQueryRotate - plugin to rotate images by any angle cross-browse with animation support",
+  "version": "2.3",
+  "author": "wilq32 <wilq32@gmail.com>",
+  "homepage": "http://jqueryrotate.com/",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/wilq32/jqueryrotate.git"
+  },
+  "keywords": [
+    "jquery",
+    "rotate",
+    "plugin",
+    "cross-browser",
+    "image",
+    "animation"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/wilq32/jqueryrotate.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "jqueryrotate!(3)*.+(js|map)"
```

Comment:
This pattern can't get any files.
Would you try glob tool and find the correct pattern?
Thanks.

Reply:
I disagree about jQueryRotate3.js it is different version of this library... we can ask author in order to add this version in separate commit as version3.0